templates/legal/open-source-licences.md

```diff
@@ -0,0 +1,300 @@
+---
```

Comment:
This page is not loading the JS fetched data:
https://canonical-com-1792.demos.haus/legal/open-source-licences?release=jammy

<img width="849" height="300" alt="Image" src="https://github.com/user-attachments/assets/ab701b89-bad0-4678-89d9-8e75c0631d94" />